dashboard/db/migrate/20170505231542_remove_name_from_plc_course.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveNameFromPlcCourse < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :plc_courses, :name
```

Comment:
So adding `:string` makes it so that I'm able to rollback, but I've still lost all of my data for this column (i.e. I end up with a column of NULL). I guess this is a question of whether it's better to allow the rollback to happen (but have lost data) or to just not allow the rollback.